### ISSUE TITLE
Idea - BAF 'Slate'/'Crib' Cards. 

### ISSUE THREAD

**Issue by ghost** [Saborknight/9liners-and-notepad#63]
**Arma 3 Version:** `x.xx` (stable / rc / dev)
**9Liner & Notepad Version:** `3.x.x` (stable / dev + commit hash)

**Mods:**
```
- CBA_A3 : 3.x.x
```

**Description:**
- I have the issued slate cards in electronic version. Would be good to get these imported as PDF or JPG or whatever is used.

**Steps to reproduce:**
- I don't know how to code myself so hoping this project is still open.

**Where did the issue occur?**
- Single player
[Slate Cards Mar 2018 AKX (2).pdf](https://github.com/Saborknight/9liners-and-notepad/files/6009469/Slate.Cards.Mar.2018.AKX.2.pdf)

**RPT log file:**
- Add a link ([gist](https://gist.github.com) or [pastebin](http://pastebin.com)) to the client and/or server RPT file.

**Comment by HTMadge** [comment]
![9liner_cas](https://user-images.githubusercontent.com/79312369/108536315-3d2e2680-72d4-11eb-88f3-4e1ee444dcb5.png)

**Comment by TheLocalPub** [comment]
I've spoken to the devs personally about this.
They were happy for me to make a SLATE card version, haven't got round to it tho.

**Comment by TheLocalPub** [comment]
Also @HTMadge Are you Royal Yeomanry?
Ex C-SQN myself.

**Comment by HTMadge** [comment]
> Also @HTMadge Are you Royal Yeomanry? Ex C-SQN myself.

Yep, C&S Sqn. Hoping this will integrate into a project we're working on atm.

**Comment by TheLocalPub** [comment]
An Arma 3 project for the RY?

I believe we've met before, I recall Madge.
I think we crossed paths on the first signal course at C-SQN and on the 2nd signal course in Bovi. 

My community has got SLATE cards in-game currently.
We don't have it in this format, in which you can use CBA binds to hotkey it open on your screen, but we've made a separate more light weight mod where in you go to the map screen, on the left hand side where you can see all the tabs, we've got our own custom tab called "Global Conflicts" (our community name), followed by different tabs such as SLATE cards, ACE Pocket medical guide, acronyms and abbreviations, and so on. 
Would this be something of interest as a work around for this?
You can quite easily edit the tabs with little .SQF knowledge.

**Comment by HTMadge** [comment]
Its a VR jackal trainer that uses steering wheels, headsets etc. Should be using VBS but I'm trialling in ARMA for now as its much faster. 
What's your name? 

Interesting, as you can imagine, the boys need a solution that is really simple to use. Like 1 button to bring up the correct slate card. So your current method might be a bit complex. I managed to get the slate cards edited in a local version but it doesnt fill out text in the right places etc. See below which was used in a brigade level demo. 
https://youtu.be/uGJloZI7r3I?t=40

**Comment by TheLocalPub** [comment]
Beezley.

Neat video. Definitely the person I thought it was.
Editing the cards to display different images is very easy.
The part that's hard is realigning the writing regions in the .SQF. Actually pretty simple, everything else is done for you from the base mod. I could probably in a few days do this if I sat down properly.

**Comment by HTMadge** [comment]
Ah yes I remember you. 
I think I got as far as you've said. But obv would rather the devs make the base model changeable so people dont have to keep downloading my modded version.

**Comment by Saborknight** [comment]
Yeah, modularity and, more importantly, making creating new formats as easy as possible has been a big focus through the development we did do. Unfortunately Arma 3 really does not make it easy on us, and with the knowledge that I've accumulated over the time since I last worked on this, I now have a better idea on how to implement this modularity.

I just need to find time to work on this again. It is something I intend to do though, because this aged mod most definitely needs the upgrade.
